README.md

```diff
@@ -16,12 +16,13 @@ By default, `kcp` only knows about:
 
 Any other resources, including standard Kubernetes resources like [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)s and the rest, can be added as CRDs and be replicated onto one or more Kubernetes clusters.
 
+The upper layer of `kcp` is a transparent multi-cluster workload service.  This service keeps track of backend "physical clusters" and syncs API resources defined in `kcp` to those physical clusters.
 
 ## Why would I want that?
 
 Kubernetes is mainly known as a container orchestration platform today, but we believe it can be even more.
 
-With the power of CRDs, Kubernetes provides a flexible platform for declarative APIs of _all types_, and the reconciliation pattern common to Kubernetes controllers is a powerful tool in building robust, expressive systems.
+Kubernetes itself can be seen as having two layers: (1) an extensible API service, and (2) a control plane built using that API service to keep track of worker nodes and manage containerized workloads on those worker nodes.  The API service of Kubernetes is extensible through two mechanisms: (i) aggregation of custom external apiservers (which _may_ be built in the same style as the `kube-apiserver` by building on [k/apiserver](github.com/kubernetes/apiserver)) and (ii) addition of CRDs.  `kcp` focuses on CRDs.  With its extensible API service, Kubernetes provides a flexible platform for declarative APIs of _all types_ --- and the reconciliation pattern common to Kubernetes controllers is a powerful tool in building robust, expressive systems.
```

Comment:
This isn't quite right, in that the foundational APIs (think Namespace and kin) still also rely on control loops external to the API server. Having _just_ an API server isn't that useful. That holds whether you want the objects in the API in order to manage container compute, a factory, home automation, or any other weird and wonderful thing you could do with an extensible CRUD API like the Kubernetes core gives you.

I recommend having a think about how to make the point here but without sacrificing accuracy over the details.